helpers/cache.js

```diff
@@ -14,10 +14,10 @@ function clear(key) {
 
 function set(key, value) {
 	if (!CACHE.hasOwnProperty(key)) {
-		debug(`cache.set: new key: key=${key}`);
+		// debug(`cache.set: new key: key=${key}`);
 		CACHE[key] = value;
 	} else {
-		debug(`cache.set: existing key: key=${key}`);
+		// debug(`cache.set: existing key: key=${key}`);
```

Comment:
Can you get rid of superfluous lines instead of commenting them out?

Reply:
done